.github/actions/cancel-workflow-run/action.yml

```
@@ -0,0 +1,71 @@
+runs:
+  using: 'composite'
+  steps:
+    - name: Cancel other running jobs
+      uses: actions/github-script@v7
+      with:
+        github-token: ${{ inputs.github-token }}
+        script: |
+          const { owner, repo } = context.repo;
+          const run_id = context.runId;
+          const currentJob = process.env.GITHUB_JOB;
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)